Custom-Audio-Driver/External-Audio-Device/OTDefaultAudioDevice.m

```diff
@@ -476,31 +478,38 @@ - (void) setupAudioSession
     
     if (error)
         OT_AUDIO_DEBUG(@"Audiosession setActive %@",error);
-    
-    [self setBluetoothAsPrefferedInputDevice];
+
+    if (@available(iOS 15, *)) {
+        // do nothing refer comments for self.setBluetoothAsPrefferedInputDevice method.
```

Comment:
Why not log this instead?  A comment may be easy to miss.

Reply:
We are making BT work the same as before, everywhere. There is no "missing" or "changed behavior" stuff to log.